web-ui/pages/index.vue

```diff
@@ -19,18 +19,33 @@ export default {
     EditorPanel,
     AppContainer,
   },
-  mounted() {
-    window.log = output => {
-      this.addToLog(output);
-    };
-  },
   data() {
     return {
       code:
-        "const a = 10;\nconst b = 20;\nconsole.log(a + b);\nconsole.log('A String');\n",
-      console: "",
+        "const a = 10;\nconst b = 20;\nconsole.log(a + b);\nconsole.log('A String');\nsdfsdf\n",
+      console: [],
     };
   },
+  mounted() {
+    if (typeof Storage !== "undefined")
+      if (localStorage.code) this.code = localStorage.code;
+
+    window.log = (output, type) => {
+      this.addToLog(output, type);
+    };
+
+    let storeCode = e => {
```

Comment:
Prefer `const` over `let` unless you explicitly need to modify the variable - which here you don't.

Also, `saveCode` would be a better name, though you can just inline this function below.